internal/core/data/interfaces/mocks/DBClient.go

```diff
@@ -2,8 +2,10 @@
 
 package mocks
 
-import "github.com/stretchr/testify/mock"
-import "github.com/edgexfoundry/go-mod-core-contracts/models"
+import go_mod_core_contractsmodels "github.com/edgexfoundry/go-mod-core-contracts/models"
```

Comment:
Use consistent aliasing to resolve the name conflicts. i.e. leave this as modes and alias models below as "correlation"

Reply:
Actually I changed this alias to `contracts` which removes the need to alias the correlation model. This has to be regenerated every time the mock changes anyway.